src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1509,9 +1509,9 @@
       "additionalProperties": false,
       "properties": {
 
-        "physicalLocation": {
+        "location": {
```

Comment:
>location [](start = 9, length = 8)

@lgolding, if we're putting the hood up here, I'd tend to wonder whether this shouldn't be an array of locations. this would allow the notification to bundle some notifications, e.g., 'Could not access one or more directory locations.

That seems fairly non-intrusive. OTOH, I don't think we should add all the relationships, related locations machinery, it is a bridge too far at this late date.

Reply:
Harlene, we're turning this into an array of locations. :)

---
In reply to: [278144207](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278144207) [](ancestors = 278144207)